lisa/sut_orchestrator/libvirt/schema.py

```diff
@@ -64,6 +64,8 @@ class BaseLibvirtNodeSchema:
     disk_img: str = ""
     # Format of the disk image specified above.
     disk_img_format: str = ""
+    # Pass disk_size_gb to resize the guest image os disk
+    disk_size_gb: Optional[int] = None
```

Comment:
Perhaps `disk_img_resize_gib`?

`disk_img` to match the names of the existing properties. `resize` to make it clear that it is resizing the image. And `gib` to mean Gibibytes (1024^3) instead of gigabytes (1000^3).

Reply:
Done